src/main/java/homework08/BringBookThread.java

```diff
@@ -0,0 +1,20 @@
+package homework08;
+
+public class BringBookThread extends Thread {
+	protected BookCrossingLibrary bcl;
```

Comment:
Why protected?

Reply:
breeding of entitys

Reply:
> breeding of entitys

what does it mean?

Reply:
> Why protected?

I see protected in example - I do protected in my homework : )